src/components/Admin/ClientList.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+import Loading from '../../shared/Animations/Loading';
+import GetUsers from '../../helpers/GetUsersListHelper';
+import { mapStateToProps } from '../../redux/actions';
+import ClientField from './ClientField';
+import { AdminPageButtons } from '../../utils/configs';
+import { PageButton } from '../Buttons';
+import MenuHelper from '../../helpers/MenuHelper';
+
+const ClientList = ({ appState, dispatch, match }) => {
+  const { client } = match.params;
+  const history = useHistory();
+  const { userId } = appState;
+
+  const onSelect = (event) => {
+    const path = MenuHelper({ event, userId });
+    history.push(path);
+  };
+
+  useEffect(() => {
+    GetUsers({ client, dispatch })
+      .then();
```

Comment:
then what?

Reply:
// eslint-disable-next-line camelcase
            id, phone_number, first_name, last_name,
          } = passenger;
          this solved on #13 PR